.buildkite/release_pipeline.yaml

```diff
@@ -30,10 +30,20 @@ steps:
       BUILD_PKG_TARGET: "x86_64-linux-kernel2"
 
   - label: ":windows: :habicat: core/hab"
-    command: /bin/true
-    skip: "No Windows builders yet"
+    command: .buildkite/scripts/build_component.ps1 -Component hab -SourceChannel unstable -BaseHabVersion "0.68.0/20181108152053"
```

Comment:
This is a result of testing - once we get a full release out with this code, we can use stable/latest.